src/containerapp/azext_containerapp/_help.py

```diff
@@ -572,6 +572,15 @@
         az containerapp auth update -g myResourceGroup --name MyContainerapp --proxy-convention Standard
 """
 
+helps['containerapp env workload-profile set'] = """
+    type: command
+    short-summary: Create or update an existing workload profile in a Container Apps environment
+    examples:
+    - name: Create or update an existing workload profile in a Container Apps environment
```

Comment:
have to be existing?

Reply:
Prefer to keep the message to the original message.
But In fact it can be not existing.

Reply:
I mean existing is not required in description